api/src/test/java/org/motechproject/nms/api/osgi/KilkariControllerBundleIT.java

```diff
@@ -272,6 +279,25 @@ public void testCreateSubscriptionRequestDifferentPacks() throws IOException, In
     }
 
     @Test
+    public void testCreateSubscriptionsNoLanguageInDB() throws IOException, InterruptedException {
+        setupData();
```

Comment:
We could probably take this call to setupData() out of here and just decore the setupData() method with @before to be run before every test